transport/RtcePoll.cpp

```diff
@@ -208,21 +221,17 @@ void RtcePollImpl::run()
 
     while (_running)
     {
```

Comment:
The wait on kernel_fd will not immediately return if monitored sockets is empty. Therefore, there is no need for sleep to avoid tight loop in case the socket list is empty.